app/controllers/comments_controller.rb

```diff
@@ -3,11 +3,11 @@ def create
     user = current_user
```

Comment:
- [ OPTIONAL ] I think it would be great if you create new controller since you put it in your routes to be somthing like this and I assign it as optional because the previous reviewer didn't mention it but I highly recommend to add it ♻💯
```ruby
  def new
    @comment = Comment.new
  end
```